apps/playground/src/components/playground/image-page-client.tsx

```
@@ -177,17 +178,34 @@ export default function ImagePageClient({
 		}
+		if (
+			!config.supportsQuality ||
+			!(config.availableQualities as readonly string[]).includes(imageQuality)
+		) {
+			setImageQuality(config.defaultQuality ?? "auto");
+		}
 		if (!isEditModel) {
 			setInputImages([]);
 		}
-	}, [selectedModels, imageSize, imageGenModels, isEditModel]);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)